README.md

```diff
@@ -67,7 +67,7 @@ CREATE TABLE "Blogs" (
 );
 ```
 
-Most of the attributes make use of database regular expressions. For SQL Server, this requires some initial setup - [follow these docs](https://www.red-gate.com/simple-talk/sql/t-sql-programming/tsql-regular-expression-workbench). Note that processing complex regular expressions does have a cost, so consider performance before turning these constraints on for write-intensive applications.
+Most of the attributes make use of database regular expressions. For SQL Server databases with compatibility level 160 or lower, this requires some initial setup - [follow these docs](https://www.red-gate.com/simple-talk/sql/t-sql-programming/tsql-regular-expression-workbench). Note that processing complex regular expressions does have a cost, so consider performance before turning these constraints on for write-intensive applications.
```

Comment:
I'd maybe include a bit more info here: things just work if you use UseAzureSql() (we should mention this), but if you're using UseSqlServer (2025 preview), you need to tell EF explicitly that you're on compatibility level 170 (i.e. it's not just what SQL Server you're using or what compatibility level it has configured, it's also what you tell EF). I'd include a code sample showing UseCompatibilityLevel for the on-prem SQL Server 2025 case.